storybook/native/main.js

```diff
@@ -22,7 +22,9 @@ configure(() => {
 
 // add decorators
 const SpaceAround = styled.View`
-  margin: 20px;
+  padding: 20px;
+  height: 100%;
+  width: 100%;
```

Comment:
We need to implement `Size` component for these use-cases. @kamleshchandnani

Reply:
Yes already on roadmap sir for next sprint